src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java

```diff
@@ -84,9 +89,11 @@ public InitCodeView generateRequestObjectInitCode(
             .valueGenerator(valueGenerator)
             .initStructure(initFieldStructure)
             .method(context.getMethod())
+            .initObjectType(context.getMethod().getInputType())
+            .initObjectName(Name.from("request"))
```

Comment:
Be consistent with ordering - everywhere else has 'initObjectName' before 'initObjectType'